pipewire.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "pipewire",
+  "buildsystem": "meson",
+  "config-opts": [
+    "-Daudiotestsrc=disabled",
+    "-Droc=disabled",
+    "-Dvideotestsrc=disabled",
+    "-Dvolume=disabled",
+    "-Dvulkan=disabled",
+    "-Ddocs=disabled",
+    "-Dman=disabled",
+    "-Dbluez5-codec-ldac=disabled",
+    "-Dbluez5-codec-aptx=disabled",
+    "-Dlibcamera=disabled",
+    "-Dudevrulesdir=/app/lib/udev/rules.d/",
+    "-Dsession-managers=[]",
+    "-Dtests=disabled",
+    "-Dexamples=disabled",
+    "-Dpw-cat=disabled"
+  ],
+  "cleanup": [
+    "/include",
+    "/bin"
+  ],
+  "sources": [
+    {
+      "type": "git",
+      "url": "https://github.com/pipewire/pipewire.git",
+      "tag": "0.3.40",
```

Comment:
This should be wired up to [the external data checker](https://github.com/flathub/flatpak-external-data-checker), otherwise it will almost definitely fall behind on bug fix releases and such.

Reply:
I've added the x-checker-data block. is there a way to confirm i did it correctly?

Reply:
You can test it by setting an old version/tag for the module and running f-e-d-c (flatpak-external-...). The quickest way to get f-e-d-c is to [install it from Flathub](https://github.com/flathub/flatpak-external-data-checker/#installation).

Don't use the git checker, AFAIK it has an issue with fetching timestamp, and the end result is that it will open an update PR every time it runs if the previous one wasn't merged and applied the changes.

You can use Anitya checker like this
```
"x-checker-data": {
    "type": "anitya",
    "project-id": 57357,
    "stable-only": true,
    "tag-template": "$version"
}
```
You can also use JSON checker with GitLab's API
```
"x-checker-data": {
    "type": "json",
    "url": "https://gitlab.freedesktop.org/api/v4/projects/pipewire%2Fpipewire/repository/tags",
    "tag-query": "first(.[].name) ",
    "version-query": "$tag | sub(\"^v\"; \"\")",
    "commit-query": ".[] | select(.name==$tag) | .commit.id",
    "timestamp-query": ".[] | select(.name==$tag) | .commit.created_at"
}
```

Reply:
Thanks. Using anitya now and tested with f-e-d-c.